cli/cmd/capture/create.go

```diff
@@ -260,6 +311,20 @@ func createCaptureF(kubeClient kubernetes.Interface) (*retinav1alpha1.Capture, e
 		capture.Spec.OutputConfiguration.BlobUpload = &secretName
 	}
 
+	if s3Bucket != "" {
+		secretName, err := createSecretFromS3Upload(kubeClient, s3AccessKeyID, s3SecretAccessKey, name)
+		if err != nil {
+			return nil, fmt.Errorf("failed to create s3 upload secret: %w")
```

Comment:
Whoops, looks like the err was dropped
```suggestion
			return nil, fmt.Errorf("failed to create s3 upload secret: %w", err)
```

Reply:
Thanks for catching that. updated.